cocos/editor-support/cocostudio/ActionTimeline/CCSkeletonNode.cpp

```diff
@@ -183,14 +184,18 @@ void SkeletonNode::visit(cocos2d::Renderer *renderer, const cocos2d::Mat4& paren
             (*it)->visit(renderer, _modelViewTransform, flags);
     }
 
-    if (visibleByCamera)
+    checkSubBonesDirty();
+    for (const auto& bone : _subOrderedAllBones)
     {
-        this->draw(renderer, _modelViewTransform, flags);
-        // batch draw all sub bones
-        _batchBoneCommand.init(_globalZOrder, _modelViewTransform, parentFlags);
-        _batchBoneCommand.func = CC_CALLBACK_0(SkeletonNode::batchDrawAllSubBones, this, _modelViewTransform);
-        renderer->addCommand(&_batchBoneCommand);
+        visitSkins(renderer, bone);
```

Comment:
because i don't want to public `visitSkins`, and `BoneNode` is not a friend of `SkeletonNode`.